doc/concepts/configuration/configuration_etcd.rst

```diff
@@ -6,92 +6,150 @@ Storing configuration in etcd
 ..  admonition:: Enterprise Edition
     :class: fact
 
-    Centralized configuration is supported by the `Enterprise Edition <https://www.tarantool.io/compare/>`_ only.
+    Storing configuration in etcd is supported by the `Enterprise Edition <https://www.tarantool.io/compare/>`_ only.
 
-..  TODO
-    https://github.com/tarantool/doc/issues/3658
+Tarantool enables you to store configuration data in one place using etcd.
+To achieve this, you need to define how to access etcd and put a :ref:`YAML configuration <configuration_file>` to an etcd server.
 
-    - Install and configure etcd (authentication, TLS)
-    - Local etcd configuration (mention env vars)
-        - endpoints
-        - key prefix
-        - auth
-        - TLS
-        - http (timeout, socket)
-    - Put a remote config
-        - etcdctl put
-        - tt cluster publish
-    - Show cluster config
-        - etcdctl get
-        - tt cluster show
-    - Start app
-        - Local config
-        - Env vars
-    - Reload config
-        - auto
-        - manual (config.reload)
 
+.. _etcd_local_configuration:
 
-    Local config (``config.yaml``):
+Local etcd configuration
+------------------------
 
-    ..  literalinclude:: /code_snippets/test/config/etcd.yaml
-        :language: yaml
-        :dedent:
+To store a cluster's configuration in etcd, you need to provide etcd connection settings in a local configuration file.
+These settings are used to :ref:`publish <etcd_publishing_configuration>` a cluster's configuration and :ref:`show <etcd_showing_configuration>` it.
```

Comment:
As I see, here we describe some layout that allows several scenarios:

* Store and load configuration in etcd using `tt`.
* Start tarantool instance with loading the configuration from etcd.

While `tt` assumes that `config.yaml` is always present (and named `config.yaml`, not in some other way), tarantool has more flexibility here.

First, it allows arbitrary config file name (specified using `--config` or `TT_CONFIG`).

Second, it may be configured using etcd without any local configuration file. `TT_CONFIG_ETCD_ENDPOINTS` and `TT_CONFIG_ETCD_PREFIX` must be set in the case.

I don't know, whether it is important for this article, so I'll just let you know and let you decide.

Reply:
Thanks for catching this. The main source of info about starting instances is a corresponding section in the `Configuration` topic:
https://docs.d.tarantool.io/en/doc/3.0-config-etcd/concepts/configuration/#starting-tarantool-instances

But we definitely need to mention some advanced capabilities that the `tarantool` command provides. Will extend this section a bit:
https://docs.d.tarantool.io/en/doc/3.0-config-etcd/concepts/configuration/configuration_etcd/#starting-tarantool-instances